middleware/kubernetes/README.md

```diff
@@ -0,0 +1,238 @@
+Kubernetes Middleware
```

Comment:
Useful docs! But can you try to adhire to the format of the other READMEs? I do agree that most can be extended. The original README from SkyDNS must also be added to the etcd middleware.